### DISCUSSION TITLE
Storefront CORS with preview envs in nextjs

### DISCUSSION THREAD

**Discussion by fxmb** [medusajs/medusa#2760] [Q&A] [answered]
What is the approach with CORS on preview envs on nextjs? 
I see no way to test the store in preview envs with unique URLs given CORS errors. Any current ways of solving this?

**Comment by srindom** [comment, marked as answer]
We have a PR up that will allow you to specify your CORS configuration using a Regex: https://github.com/medusajs/medusa/pull/1935

Once that is merged you should be able to construct a regex that allows the preview URLs to get access.
